base_local_planner/src/speed_limiters/static_object_speed_limiter.cpp

```diff
@@ -99,7 +99,7 @@ namespace base_local_planner
       return true;
     }
 
-    if (const char *env_var = std::getenv("ROSEVN_STATIC_OBJECT_LIMITER_ENABLED"))
+    if (const char *env_var = std::getenv("ROSENV_STATIC_OBJECT_LIMITER_ENABLED"))
```

Comment:
we shouldn't be reading env vars to set configs. this should be taken from the config file specified here: https://github.com/6RiverSystems/navigation/blob/d7fa1fc7f188d421d011187cdad40a98e6698670/base_local_planner/cfg/StaticObjectSpeedLimiter.cfg

The env var should be passed into the config file similar to https://github.com/6RiverSystems/ros/blob/master/srsnode_navigation/launch/speed_limiter_params.launch#L4


This will make it WAY easier to determine if the config was set by looking at the dynamic reconfigure or the rosparam dump.

Reply:
Aren't the configs read and processed at compile time?  or are they reprocessed after mfpCommander receives the site config?
Or do you suggest reading the site config and modifying the config from mfpCommander?

Reply:
These are python, to read the env var it would need to do something like this 
`#!/usr/bin/env python
# Static Object Speed Limiter configuration

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, double_t, int_t, bool_t, os
gen = ParameterGenerator()

gen.add("enabled", bool_t, 0, "Is this speed limiter enabled for the current chuck.", os.environ['ROSENV_STATIC_OBJECT_LIMITER_ENABLED'])`

Reply:
You can pass the env through the launch files similar to how we do everything else. Something like this would do the trick
https://github.com/6RiverSystems/navigation/pull/165
https://github.com/6RiverSystems/ros/pull/1730